kernel-kit/build.sh

```diff
@@ -197,7 +197,7 @@ FW_URL=${FW_URL:-http://distro.ibiblio.org/puppylinux/firmware}
 # $package_name_suffix $custom_suffix $kernel_ver
 kernel_version_full=${kernel_version}${custom_suffix}
 kernel_srcsfs_version=${kernel_version}
-aufs_utils_git="git://git.code.sf.net/p/aufs/aufs-util.git"
+aufs_utils_git="https://git.code.sf.net/p/aufs/aufs-util"
```

Comment:
For these rare chases may be better to add ```aufs_utils_git_alt="https://git.code.sf.net/p/aufs/aufs-util"``` and then change the code [later](https://github.com/peabee/woof-CE/blob/f630ebd1248bdb7720cda73824f3015f43f484f9/kernel-kit/build.sh#L645) to
```
if [ ! -d aufs-util ] ; then
    git clone ${aufs_utils_git} aufs-util
        if [ $? -ne 0 ] ; then
            git clone ${aufs_utils_git_alt} aufs-util
                 if [ $? -ne 0 ] ; then
			rm -rf aufs-util
			exit_error "Failed to get aufs-util from git, do it manually. Kernel is compiled OK :)"
		fi
	fi
fi
```
The only problem in this case is that you may end up with a different aufs-utils if there is any other reason for git to be inaccessible.